ui/components/app/modals/contract-details-modal/index.scss

```diff
@@ -0,0 +1,99 @@
+.contract-details-modal {
+  background-color: var(--color-background-default);
+  width: 360px !important;
+
+  &__content {
+    padding: 24px 16px 32px 16px;
+    border-bottom: 1px solid var(--color-border-muted);
+
+    &__header {
+      font-style: normal;
+      font-weight: 700;
+      font-size: 16px;
+      line-height: 24px;
+      display: flex;
+      align-items: center;
+    }
```

Comment:
All of this should be achieveable with our <Typography> component and the boxProps on that component. When translating styles from figma we need to make sure we are translating to our design system components and standards as best as possible. For documentation see storybook https://metamask.github.io/metamask-storybook/index.html?path=/story/getting-started-introduction--page.

Reply:
Hi @brad-decker, You can see changes here: [d1ae5af](https://github.com/MetaMask/metamask-extension/pull/15549/commits/d1ae5af192efb5c81d7eefe7c028e6a1223c6e37)  and [5e6a58a](https://github.com/MetaMask/metamask-extension/pull/15549/commits/5e6a58ac132773166eebd4c425cde0751a724c06)